### PR TITLE
Upgrade bridge pangolin-pangoro

### PR DESCRIPTION
**Pull request by fewensa**
Related:
- https://github.com/darwinia-network/parity-bridges-common/pull/12
- https://github.com/paritytech/parity-bridges-common/pull/1323

### CODE REVIEW

File: bridges/pangolin-pangoro/client-pangoro/src/pangoro.rs [hunk 1]
```diff
@@ -42,6 +55,30 @@ impl Chain for PangoroChain {
...
+        drml_bridge_primitives::TO_PANGORO_LATEST_RECEIVED_NONCE_METHOD;
+    const FROM_CHAIN_LATEST_RECEIVED_NONCE_METHOD: &'static str =
+        drml_bridge_primitives::FROM_PANGORO_LATEST_RECEIVED_NONCE_METHOD;
+    const FROM_CHAIN_LATEST_CONFIRMED_NONCE_METHOD: &'static str =
+        drml_bridge_primitives::FROM_PANGORO_LATEST_CONFIRMED_NONCE_METHOD;
+    const FROM_CHAIN_UNREWARDED_RELAYERS_STATE: &'static str =
+        drml_bridge_primitives::FROM_PANGORO_UNREWARDED_RELAYERS_STATE;
+    const PAY_INBOUND_DISPATCH_FEE_WEIGHT_AT_CHAIN: Weight =
+        drml_bridge_primitives::PAY_INBOUND_DISPATCH_FEE_WEIGHT;
+    const MAX_UNREWARDED_RELAYERS_IN_CONFIRMATION_TX: MessageNonce =
```
Comment (fewensa) [inline bridges/pangolin-pangoro/client-pangoro/src/pangoro.rs:75]: https://github.com/paritytech/parity-bridges-common/blob/ddb4517e133c6d194d42733b659ff7624e22133e/relays/client-millau/src/lib.rs#L70-L73

**Comment by fewensa** [comment]
```
2022-02-22 03:27:16  INFO bridge: Synced Some(1) of Some(2) nonces in Pangoro::MessagesDelivery -> Pangolin::MessagesDelivery race
2022-02-22 03:27:16  INFO bridge: Synced Some(0) of Some(0) nonces in Pangolin::MessagesDelivery -> Pangoro::MessagesDelivery race
2022-02-22 03:27:16  INFO bridge: Synced Some(0) of Some(0) nonces in Pangoro::ReceivingConfirmationsDelivery -> Pangolin::ReceivingConfirmationsDelivery race
2022-02-22 03:27:16  INFO bridge: Synced Some(1) of Some(1) nonces in Pangolin::ReceivingConfirmationsDelivery -> Pangoro::ReceivingConfirmationsDelivery race
2022-02-22 03:27:16  INFO client-pangoro: [pangoro] You are assigned relayer, you must be relay this nonce(2)
2022-02-22 03:27:16  INFO client-pangoro: [pangoro] About nonce 2 decide is true
2022-02-22 03:27:16  INFO bridge: Synced 355 of 355 headers
2022-02-22 03:27:16  INFO bridge: Synced 354 of 354 headers
```

File: bridges/pangolin-pangoro/client-pangoro/src/pangoro.rs (hunk 1 above)
Comment (HackFisher) [reply bridges/pangolin-pangoro/client-pangoro/src/pangoro.rs:75]: https://github.com/paritytech/parity-bridges-common/pull/1234

Comment (fewensa) [reply bridges/pangolin-pangoro/client-pangoro/src/pangoro.rs:75]: https://github.com/darwinia-network/parity-bridges-common/blob/55b88ea4a0a675535dc05cf17c0b027df999524a/relays/lib-substrate-relay/src/messages_lane.rs#L199

https://github.com/darwinia-network/parity-bridges-common/blob/55b88ea4a0a675535dc05cf17c0b027df999524a/relays/messages/src/message_lane_loop.rs#L325

https://github.com/darwinia-network/parity-bridges-common/blob/55b88ea4a0a675535dc05cf17c0b027df999524a/relays/messages/src/message_race_delivery.rs#L69

https://github.com/darwinia-network/parity-bridges-common/blob/55b88ea4a0a675535dc05cf17c0b027df999524a/relays/messages/src/message_race_delivery.rs#L446


--

https://github.com/darwinia-network/parity-bridges-common/blob/55b88ea4a0a675535dc05cf17c0b027df999524a/relays/messages/src/message_race_delivery.rs#L480

**Review by HackFisher — APPROVED** [review]

**Pull request merged**